doc/install.rst

```diff
@@ -79,8 +79,25 @@ systems. Once docker is installed, just pull the latest image with::
 
 and follow the commands on the `image-website <https://hub.docker.com/r/flatironinstitute/triqs>`_.
 
-.. note:: Docker images allow for an easy set-up on cluster machines that provide the virtualization software `singularity <https://www.sylabs.io/guides/latest/user-guide/>`_.
-          For more information on the use of singularity in conjunction with docker images please refer to the `documentation <https://www.sylabs.io/guides/latest/user-guide/singularity_and_docker.html>`_.
+.. _triqs_singularity:
+Singularity
+-----------
+
+The virtualization software `singularity <https://www.sylabs.io/guides/latest/user-guide/>`_ allows for easy set-up on cluster machines.
+Ask your cluster administator to install the singularity software. The triqs setup is then as simple as::
+
+        singularity pull docker://flatironinstitute/triqs
+
+which will generate an image file `triqs_latest.sif`. You can then execute commands inside this image file, e.g::
+
+        singularity exec triqs_latest.sif python -c "from pytriqs import *"
+
+In order to run your code in parallel you will need to use OpenMPI version 2.1.1 to match the version of the singularity image.
```

Comment:
OpenMPI > Open MPI (Not a big change but I think it's OpenMP and Open MPI, might help people to distinguish if there's a typo in the way you wrote it)